src/main/java/view/InstructionsView.java

```diff
@@ -172,6 +172,7 @@ public void actionPerformed(ActionEvent e) {
                 ActionEvent.ACTION_PERFORMED, null));
     }
 
+    // This creates the visual examples for the user.
     private JPanel createExample(String word, int highlightedIndex) {
```

Comment:
this is not really java doc its more like a comment, what you had up there in SwitchToProfileView is proper JavaDoc